lib/govspeak/post_processor.rb

```diff
@@ -94,7 +94,9 @@ def self.extension(title, &block)
       document.css(":not(thead) tr td:first-child").map do |el|
```

Comment:
The PR description is talking about this problem being to do with table headers, but this selector seems to be specifically not table headers, can you explain?

Reply:
This is targetting table headers that exist outside of the table head. They appear in the left hand column and have a scope of "row". The ones in the table head are dealt with elsewhere (Kramdown, I think) and we had to add this extension to allow row headers.

Reply:
Ah, I think the confusion is due to the overlap when talking about "table headers" - the difference between `thead` and `th` elements. What's the element being targeted here? This line appears to refer to the first `td` in a `tr` that is not inside a `thead` - is that correct?

Reply:
Yes, that's right, <th> outside the <thead> in the first column with a scope of row.